lib/components/component.dart

```diff
@@ -72,6 +78,9 @@ abstract class Component {
   /// Called right before the component is removed from the game
   void onRemove() {}
 
+  /// Called to check whether the point should be counted as a tap on the component
```

Comment:
Should we add some more info here that it has a placeholder implementation by default that always returns false and that other components should override this to "make it real"?

Reply:
Yeah definitely, and doc page for it.